Tools/LogAnalyzer/tests/TestOptFlow.py

```diff
@@ -113,7 +114,7 @@ def WARN():
                     startRollIndex = i
                     break
 
-            # calculate the end time for the roll calibration
+            # calculate the end time for the pitch calibration
```

Comment:
Sorry Paul, I gave you a bum steer here.  I deleted the comment seconds after I created it, but you picked it up from email, I assume.

I think this *is* roll, isn't it?